setup.py

```diff
@@ -36,16 +30,17 @@
                    'Programming Language :: Python :: 3.6',
                    'Programming Language :: Python :: 3.7',
                    'Programming Language :: Python :: 3.8',
```

Comment:
Can we shift the Python versions to the 3.8 - 3.11 range?

Also, these metadata have to be updated because below `python_requires='>=3.8',`

Do we test all these python versions?

Reply:
we do not, missed those lines, mkaing it 3.8-3.11